canonical/ubuntu.com
# Canonical CLA integration

## Done

- [List of work items including drive-bys - remember to add the why and what of this work.]

## QA

- Check out this feature branch
- Run the project locally `dotrun`
- Go to http://0.0.0.0:8001/legal/contributors/agreement
- **Individual agreement**:
  - Make sure that you haven't signed the CLA yet (should see value `false`): `curl https://cla.staging.canonical.com/cla/check?emails=YOUR_EMAIL`
  - Choose the individual agreement type
  - Fill the informations form
  - Sign in using GitHub and choose an email address
  - Submit the form
  - Make sure that your chosen email is signed now: `curl https://cla.staging.canonical.com/cla/check?emails=YOUR_EMAIL`
  - Refill the form and choose a different email address with Launchpad
  - Make sure the CLA check is working
- **Organisation agreement**:
  - Please contact me so I can change the community contact email so you can receive the email instead, this is necessary to approve the organisation once the form has been submitted
  - Fill the organisation contact form
  - Choose an email from GitHub or Launchpad,  the email domain shouldn't be GMail or other email provider, something like Canonical would work
  - Make sure that you have received a confirmation email
  - Make sure that you have received an email with subject `Canonical CLA signed - Action Required`
  - Click on the `Manage ... org CLA`
  - Make sure that you can view the organisation's management UI

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
